src/main/resources/jsv-messages_es.properties

```diff
@@ -0,0 +1,70 @@
+$ref = {0}: tiene un error con ''refs''
```

Comment:
The `encoding` test in `LocaleTest` needs to be modified to ensure that the encoding of this file works for Java 8.

Reply:
this is taken care

Reply:
Could you please le me know how to do that?

Reply:
You need to modify this test.

https://github.com/networknt/json-schema-validator/blob/ffec188f5d0b5d4f1190a56f8cbfb48b4c15a89d/src/test/java/com/networknt/schema/LocaleTest.java#L118-L125

You also need to modify this to add the supported locale

https://github.com/networknt/json-schema-validator/blob/ffec188f5d0b5d4f1190a56f8cbfb48b4c15a89d/src/main/java/com/networknt/schema/i18n/Locales.java#L33-L35